README.md

```diff
@@ -1,30 +1,27 @@
-# ruby_rbenv Chef Cookbook
+# Chef ruby_rbenv Cookbook
 
 [![Build Status](https://travis-ci.org/sous-chefs/ruby_rbenv.svg?branch=master)](https://travis-ci.org/sous-chefs/ruby_rbenv) [![Cookbook Version](https://img.shields.io/cookbook/v/ruby_rbenv.svg)](https://supermarket.chef.io/cookbooks/ruby_rbenv)
 
 ## Description
 
-Manages [rbenv][rbenv_site] and its installed Rubies. Several lightweight resources and providers ([LWRPs][lwrp]) are also defined.
+Manages [rbenv][rbenv_site] and its installed Rubies.
 
-**WARNING**: As of version 1.0 this cookbook was renamed from rbenv to ruby_rbenv so it could be uploaded to the Supermarket. Attributes and resources (LWRPs) retain the rbenv namespace for compatibility, but if you wrap this cookbook you'll need to update the recipes you include. Use 0.9.0 if you need the existing name.
+Several custom_resources are defined facilitate this.
 
-## Requirements
+**WARNING**: As of version 1.0 this cookbook was renamed from rbenv to ruby_rbenv so it could be uploaded to the Supermarket.
 
-### Platforms
+## Requirements
 
-The following platforms have been tested with this cookbook, meaning that the recipes and LWRPs run on these platforms without error:
+### Chef
 
-- Arch
-- Debian derivitives (Debian 7+, Ubuntu 12.04+, Linux Mint)
-- Freebsd
-- Fedora
-- Gentoo
-- Mac OS X
-- RHEL deritivites (RHEL, CentOS, Amazon Linux, Oracle, Scientific Linux)
+This cookbook requires Chef 12.9+.
 
-Please [report][issues] any additional platforms so they can be added.
+### Platform
 
-### Chef
+- Debian derivatives (Debian 7+, Ubuntu 12.04+, Linux Mint)
```

Comment:
We should remove the mention of Ubuntu 12.04. Also who knows if linux mint works. It probably does, but we're not testing it